aries-site/src/pages/foundation/voice-and-tone.mdx

```diff
@@ -460,10 +458,12 @@ Problems:
 ### Version 4
 
 <ClusterExample>
+  <Box width="100%">
   <Text weight="bold">
     The cluster <Anchor label="ABC" /> is using over 95% of its memory (42GB of
     free memory out of 1TB total).
   </Text>
+  </Box>
 </ClusterExample>
 
 Problems:
```

Comment:
Lines 562 & 564 are using the incorrect colors. Should be `icon-info` and `icon-critical`. Since we need to touch this, these ClusterExamples should be replaced with Notification.

Reply:
Done